curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646e44ec9bd212021dc7de44.md

```diff
@@ -0,0 +1,162 @@
+---
+id: 646e44ec9bd212021dc7de44
+title: Step 46
+challengeType: 0
+dashedName: step-46
+---
+
+# --description--
+
+You will add the `reverse()` method to the end of `split()` method.
```

Comment:
This is also the first time we are introducing the `reverse` method so we should provide a brief explanation here.

```suggestion
The `reverse()` method is used to reverse an array in place. 

Chain the `reverse()` method to the end of `split()` method.
```

Reply:
added